public/app/features/alerting/AlertRuleList.tsx

```diff
@@ -114,7 +114,7 @@ export class AlertRuleListUnconnected extends PureComponent<Props> {
               </div>
             </div>
             <div className="page-action-bar__spacer" />
-            {config.featureToggles.ngalert && (
+            {config.unifiedAlertingEnabled && (
               <LinkButton variant="primary" href="alerting/ng/new">
                 Add NG Alert
```

Comment:
Nit: Do we want to use `NG Alert` terminology in the UI?

Reply:
most probably no!

Reply:
This bit of code is just remnants of the aborted first iteration of ngalerting. This component isn't rendered at all with unified alerting switched on, so no problem

Reply:
thanks @domasx2!

Reply:
I was thinking we'd clean all of this up when "legacy" alerting is removed entirely (original plan was to do a hard switch with 8.0), but now not sure when that will happen :)